org.mixedrealitytoolkit.input/Tests/Runtime/Utilities/InputTestUtilities.cs

```diff
@@ -108,15 +115,26 @@ public static bool IsEyeGazeTracking
         #endregion Simulated Devices
 
         /// <summary>
-        /// Creates and returns the MRTK rig.
+        /// Creates and returns the deprecated pre-XRI3 MRTK rig.
         /// </summary>
+        [Obsolete("Deprecated, please use InstantiateRigForXRI3()")]
         public static GameObject InstantiateRig()
         {
             Object rigPrefab = AssetDatabase.LoadAssetAtPath(MRTKRigPrefabPath, typeof(Object));
             rigReference = Object.Instantiate(rigPrefab) as GameObject;
             return rigReference;
         }
 
+        /// <summary>
+        /// Creates and returns the MRTK rig for XRI3+.
+        /// </summary>
+        public static GameObject InstantiateRigForXRI3()
```

Comment:
`InstantiateControllerlessRig`

Reply:
Done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/757/commits/dcbfb55bc7fb75e8ad8333ee8a2dab3b7e524a40 , resolving.